nevergrad/parametrization/utils.py

```diff
@@ -152,51 +151,69 @@ def __call__(self, *args: tp.Any, **kwargs: tp.Any) -> str:
         return stdout
 
 
-def _make_crossover_sequence(num_sections: int, num_individuals: int, rng: np.random.RandomState) -> tp.List[int]:
-    assert num_individuals > 1
-    indices = rng.permutation(num_individuals).tolist()
-    while len(indices) < num_sections:
-        new_indices = rng.permutation(num_individuals).tolist()
-        if new_indices[0] == indices[-1]:
-            new_indices[0], new_indices[-1] = new_indices[-1], new_indices[0]
-        indices.extend(new_indices)
-    indices = indices[:num_sections]
-    if 0 not in indices:
-        indices[rng.randint(num_sections)] = 0  # always involve first element
-    return indices  # type: ignore
+class Crossover:
+    """Operator for merging part of an array into another one
+
+    Parameters
+    ----------
+    axis: None or int or tuple of ints
+        the axis (or axes) on which the merge will happen. This axis will be split into 3 parts: the first and last one will take
+        value from the first array, the middle one from the second array.
+    max_size: None or int
+        maximum size of the part taken from the second array. By default, this is at most around half the number of total elements of the
+        array to the power of 1/number of axis.
```

Comment:
@teytaud do you agree this is a sensible default?

Reply:
i have abolutely no idea... the most classical thing, for historical reasons, is to use k1 and k2 random uniform and split accordingly, but some recent excellent papers have shown that small values lead to excellent results on images.

anyway don't stop for that, we'll have to play with those values later...